From 687e5a6598f89b97c89afb94d6ac3ec024dc08e2 Mon Sep 17 00:00:00 2001
From: "Song.Li" <Song.Li@windriver.com>
Date: Thu, 26 Jul 2012 14:19:46 +0800
Subject: [PATCH] guile:meta/Makefile change install-data-hook to install-exec-hook

In guile meta/Makefile, what install-data-hook do
must be run after install-binSCRIPTS.so it's a exec hook, not a data hook.
If keep these exec-hook in data-hook,when we run make with multi-jobbing ,
install-data-hook and install-binSCRIPTS will be separately
processed in different thread, no any dependence.
that will cause error sometimes.

[YOCTO #2796]

Upstream Status:Submitted
Upstream Bugid :12015 email:12015@debbugs.gnu.org
guile bug maillist: bug-guile@gnu.org

Signed-off-by: Song.Li <Song.Li@windriver.com>
---
 meta/Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta/Makefile.am b/meta/Makefile.am
index f26fc44..e603e5c 100644
--- a/meta/Makefile.am
+++ b/meta/Makefile.am
@@ -27,7 +27,7 @@ EXTRA_DIST= \
   guild.in guile-config.in
 
 # What we now call `guild' used to be known as `guile-tools'.
-install-data-hook:
+install-exec-hook:
 	cd $(DESTDIR)$(bindir) && rm -f guile-tools$(EXEEXT) && \
 	$(LN_S) guild$(EXEEXT) guile-tools$(EXEEXT)
 
-- 
1.7.9.5

